README.md

```diff
@@ -22,7 +22,7 @@
 
 # 📖 Portfolio <a name="about-project"></a>
 
-- This website shows a sneak peek into the projects I have developed and deployed, showcasing my skills in HTML and CSS.With a passion for web development, I have honed my expertise in these technologies to create stunning and functional websites.
+- This website shows a sneak peek into the projects I have developed and deployed, showcasing my skills in HTML and CSS.With a passion for web development, I have honed my expertise in these technologies to create stunning and functional websites. You can find a live version of the website [here](https://arch-noize.github.io/portfolio-microverse/)!
```

Comment:
Nice job deploying your project👍. But now you have deployed your project, I suggest you add the live demo section back to the readme and add the link to the deployed project to the live demo section😉😉.